### PR TITLE
feat: [ANDROSDK-1897] Create new properties for map configuration 

### PR DESCRIPTION
**Pull request by taridepaco**
Related task [ANDROSDK-1897](https://dhis2.atlassian.net/jira/software/c/projects/ANDROSDK/boards/93?selectedIssue=ANDROSDK-1897)

[ANDROSDK-1897]: https://dhis2.atlassian.net/browse/ANDROSDK-1897?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: core/src/androidTest/java/org/hisp/dhis/android/testapp/settings/AppearanceSettingsObjectRepositoryMockIntegrationShould.kt [hunk 1]
```diff
@@ -139,6 +139,34 @@ class AppearanceSettingsObjectRepositoryMockIntegrationShould :
         assertThat(program2Setting?.itemHeader()?.programIndicator()).isNull()
     }
 
+    @Test
+    fun should_return_minimumLocationAccuracy_settings() {
+        val setting = d2.settingModule().appearanceSettings().getGlobalProgramConfigurationSetting()
+        assertThat(setting?.minimumLocationAccuracy()).isEqualTo(2)
```
Comment (andresmr) [inline core/src/androidTest/java/org/hisp/dhis/android/testapp/settings/AppearanceSettingsObjectRepositoryMockIntegrationShould.kt:145]: I think that minimum location accuracy it's 5

File: core/src/main/java/org/hisp/dhis/android/core/settings/ProgramConfigurationSetting.java [hunk 2]
```diff
@@ -92,6 +98,10 @@ public abstract static class Builder {
 
         public abstract Builder itemHeader(ProgramItemHeader itemHeader);
 
+        public abstract Builder minimumLocationAccuracy(Integer minimumLocationAccuracy);
+
+        public abstract Builder disableManualLocation(Boolean disableManualLocation);
```
Comment (andresmr) [inline core/src/main/java/org/hisp/dhis/android/core/settings/ProgramConfigurationSetting.java:103]: Does this property a default value or is it nullable? Should it has a default value?

Comment (marcamsn) [reply core/src/main/java/org/hisp/dhis/android/core/settings/ProgramConfigurationSetting.java:103]: I think it should be enabled by default.

Comment (vgarciabnz) [reply core/src/main/java/org/hisp/dhis/android/core/settings/ProgramConfigurationSetting.java:103]: Most of the properties here are nullable and are not given a default value in the SDK. They have a default initial value in the ASWA (the initial config) and the application gives them a default value when they are null. I would keep the same behavior for this property. 
We could consider that the SDK should be the responsible for setting the default value, but if we do so, it should be done for all the properties. So I would keep this property nullable for now

**Review by vgarciabnz — APPROVED** [review]

**Pull request merged**